cpu/stm32/periph/rtc_mem.c

```diff
@@ -104,7 +104,8 @@ void rtc_mem_write(unsigned offset, const void *data, size_t len)
         return;
     }
 
-    /* get the correct pointer */
+    uint8_t *in = (uint8_t *)data;
```

Comment:
better keep that
```suggestion
    const uint8_t *in = data;
```

Reply:
It will lead to a cast in the memcpy call, is that better?

Reply:
but the second argument of `memcpy` is `const`

Reply:
duh

Reply:
I'll addressm Kconfig now matches, do you mind if I squash once the const is added?

Reply:
done